static/app/views/acceptProjectTransfer/index.tsx

```diff
@@ -42,8 +42,13 @@ class AcceptProjectTransfer extends DeprecatedAsyncView<Props, State> {
       },
       success: () => {
         const orgSlug = formData.organization;
-
-        this.props.router.push(normalizeUrl(`/organizations/${orgSlug}/projects/`));
+        const projectSlug = this.state?.transferDetails?.project.slug;
+        const sentryUrl = ConfigStore.get('links').sentryUrl;
+        if (projectSlug === null) {
+          window.location.href = `${sentryUrl}/organizations/${orgSlug}/projects/`;
```

Comment:
@malwilley is this the right way of updating the current URL?

Reply:
It usually isn't, but since we need to change subdomains it has to be done like this to get a full page load

Reply:
You may want to add a comment explaining this actually @ceorourke so it doesn't get changed back